app/controllers/admin/brands_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Admin::BrandsController < Admin::BaseController
+  before_action :find_brand, only: %i(update destroy edit)
+
+  def index
+    @brands = Brand.page(params[:page]).per Settings.brands_controller.item_op
```

Comment:
đặt lại tên nhé, item_op là cái gì zợ ?
ko dùng tên controller để đặt tên key

Reply:
item of page ạ